website/public/js/controllers/notificationCtrl.js

```diff
@@ -53,6 +60,19 @@ habitrpg.controller('NotificationCtrl',
        Notification.mp(mana);
     });
 
+    $rootScope.$watch('user.stats.lvl', function(after, before) {
```

Comment:
I tried to write a test for this, but the `$watch` function didn't seem to run upon a `scope.$digest()` during tests.